### ISSUE TITLE
Looking for new maintainer

### ISSUE THREAD

**Issue by grantjenks** [grantjenks/py-tree-sitter-languages#54]
I no longer wish to maintain the project.

Is someone else interested in doing so?

Nothing is wrong with the project or me. I simply don’t have the time nor interest.

**Comment by Freed-Wu** [comment]
Can you add me to collaborators? Downstream projects [termux-language-server](https://github.com/termux/termux-language-server) need this project. Or you can transfer it to <https://github.com/termux> where [termux-language-server](https://github.com/termux/termux-language-server) is.

**Comment by Freed-Wu** [comment]
@grantjenks

Can you add me to collaborators? Downstream projects [termux-language-server](https://github.com/termux/termux-language-server) need this project. Or you can transfer it to <https://github.com/termux> where [termux-language-server](https://github.com/termux/termux-language-server) is.

**Comment by heijligers** [comment]
Just found this @grantjenks . Sorry to hear this project is no longer energizing you. As avid Aider user [Aider](https://github.com/paul-gauthier/aider) I'm eagerly waiting on @Freed-Wu's merge to add swift support to this repository and therefor to aider. Let us know how we can help.

**Comment by grantjenks** [comment]
Hey folks, I started the new job last week and am waiting to understand the open source contributors policy before I take the next step here. I expect it'll be soon (like this month).

**Comment by cmosguy** [comment]
@grantjenks can we move forward with your final decision? @Freed-Wu has made updates that the community needs.

Hope you are enjoying your new role by way 😀

**Comment by VladimirLevadnij** [comment]
@grantjenks Hello. I wanted to create a new issue and ask you to add GDScript support to your wonderful product https://github.com/PrestonKnopp/tree-sitter-gdscript so that it can be used in Aider.

But now I don’t know who to write to. Please tell me who should I contact with my request? :-)

**Comment by chenrui333** [comment]
@grantjenks any luck on this thread? thanks!

**Comment by Manamama** [comment]
Here why useful for `aider`: https://github.com/Aider-AI/aider/issues/3913#issuecomment-3178496454
